README.md

```diff
@@ -1,140 +1,103 @@
+[Search API documentation](https://docs.publishing.service.gov.uk/apis/search/search-api.html).
 
-### Creating search indexes from scratch
-(This is not necessary when restoring from a backup or replicating data into the development VM)
+You can also find some examples in the blog post:
+["Use the search API to get useful information about GOV.UK content"](https://gdsdata.blog.gov.uk/2016/05/26/use-the-search-api-to-get-useful-information-about-gov-uk-content/).
 
-To create an empty index:
+## Getting started
 
-    bundle exec rake search:create_index[<index_name>]
+The instructions will help you to get Search API running
+locally on your machine.
 
-To create an empty index for all Search API indices:
+### Dependencies
 
-    SEARCH_INDEX=all bundle exec rake search:create_all_indices
+- [Elasticsearch](https://github.com/elastic/elasticsearch) - "You Know, for Search...".
+- [Redis](https://redis.io/) - used by indexing workers.
+- [AWS Sagemaker](https://aws.amazon.com/sagemaker/) - used for [search relevancy](docs/relevancy.md)
 
-### Starting elasticsearch
-If you're running the GDS development VM you need to have elasticsearch running before running the tests or starting the application.
+### Prequisites
 
-Elasticsearch should start when you start up your dev VM, but if it doesn't, run:
+Install [govuk-docker](https://github.com/alphagov/govuk-docker)!
 
-	sudo service elasticsearch-development.development start
+govuk-docker, a wrapper around docker-compose, is the supported way
+to run Search API and its dependencies locally.
 
-### Running the test suite
+Once you have installed govuk-docker, run
 
-    bundle exec rake
+	cd ~/govuk/govuk-docker && make search-api
 
 ### Running the application
 
-If you're running the GDS development VM:
-
-    cd /var/govuk/govuk-puppet/development-vm && bundle exec bowl search-api
-
-Search API should then be available at
-[search-api.dev.gov.uk](http://search-api.dev.gov.uk/search.json?q=taxes).
+Once you have completed the prerequisites you'll be able to run
+Search API locally by running
 
-If you're not running the GDS development VM:
+	cd ~/govuk/search-api && govuk-docker up search-api-app
 
-    ./startup.sh
+This starts the Search API application and its dependencies.
 
-#### Workers
-Search API uses Sidekiq to manage its indexing workload. To run
-this in the development VM, you need to run both of these commands:
+The Search API will be running locally at [search-api.dev.gov.uk](search-api.dev.gov.uk/search).
 
-    # to start the Sidekiq process
-    bundle exec rake jobs:work
+If you run `docker ps` this will tell you that there are containers running
+for Search API, Nginx, Redis, Publishing API, and Elasticsearch.
 
-    # to start the search-api webapp
-    bundle exec mr-sparkle --force-polling -- -p 3233
+> Note: If you're not using docker, you can run `./startup.sh` to start the
+application. However, this is not officially supported.
```

Comment:
Here would be a good point to mention the other dependencies: "However, this is not officially supported, and you will need to run Elasticsearch and Redis yourself"